pyproject.toml

```diff
@@ -8,6 +8,13 @@ readme = "README.md"
 repository = "https://github.com/anancarv/python-artifactory"
 documentation = "https://github.com/anancarv/python-artifactory"
 keywords = ["artifactory"]
+classifiers = [
+  "Development Status :: 4 - Beta",
+  "Intended Audience :: Developers",
+  "Operating System :: OS Independent",
```

Comment:
In my opinion, we have to be sure that it works in different OS before writing it